packages/components/src/Badge/Badge.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { BADGE_SIZE_VALUES } from './Badge.types';
+import Badge from './Badge';
+
+const meta: Meta<typeof Badge> = {
+    component: Badge,
+    parameters: {
+        layout: 'centered',
+    },
+    tags: ['autodocs'],
```

Comment:
It's on me, I must admit :C but add at least `foundation` tag (similar as in CustomIcon or Button)